lib/Heartbeat/LED.cpp

```diff
@@ -60,9 +60,7 @@ void LED::setState(bool state) {
   }
 }
 
-bool LED::getState() {
-  return digitalRead(_pin) == _activeHigh;
-}
+bool LED::getState() { return digitalRead(_pin) == _activeHigh; }
```

Comment:
Why this change?

Reply:
Good question, It somehow slipped through.